### DISCUSSION TITLE
How to install ImageMagick in OpenShift RedHat Linux

### DISCUSSION THREAD

**Discussion by tomarsurendra** [ImageMagick/ImageMagick#2332] [Help]
I have OpenShift container where my Apache tomcat server is running. Now I want to install ImageMagick in that container. I have tried to download portable ImageMagick installer and but getting below error :
"AppImages require FUSE to run. You might still be able to extract the contents of this AppImage if you run it with the  --appimage-extract option."

Please suggest your solution to install ImageMagick in OpenShift RedHat Linux.

Thanks in Advance.

**Comment by urban-warrior** [comment]
Download/install FUSE and the ImageMagick portable app will run without complaint.
